### ISSUE TITLE
TiDB3.0.1can't excute some sql

### ISSUE THREAD

**Issue by jwongz** [pingcap/tidb#11528]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?
If possible, provide a recipe for reproducing the error.

grafana's dashboard throw exception "search failed",and logs of grafana write like this :

```
SELECT  dashboard.id,  dashboard.uid,  dashboard.title,  dashboard.slug,  dashboard_tag.term,  dashboard.is_folder,  dashboard.folder_id,  folder.uid AS folder_uid,  folder.slug AS folder_slug,  folder.title AS folder_title FROM  (   SELECT    dashboard.id   FROM    dashboard   INNER JOIN star ON star.dashboard_id = dashboard.id   WHERE    dashboard.org_id = 1   AND star.user_id = 1   ) AS ids INNER JOIN dashboard ON ids.id = dashboard.id LEFT OUTER JOIN dashboard folder ON folder.id = dashboard.folder_id LEFT OUTER JOIN dashboard_tag ON dashboard.id = dashboard_tag.dashboard_id;
ERROR 1105 (HY000): Column grafana_ucloud.dashboard.id is ambiguous
```
my grafana version is 6.0.1,and it works well on TiDB 2.1.4

2. What did you expect to see?

excute sql  success

3. What did you see instead?
ERROR 1105 (HY000): Column grafana_ucloud.dashboard.id is ambiguous

4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?

TiDB 3.0.1

**Comment by winkyao** [comment]
@winoros PTAL

**Comment by winoros** [comment]
@jwongz 
Sorry for your inconvenience. This bug is fixed in next minor release of 3.0, which is expected to be released at 8.6.
If this SQL can be rewritten by hand. You can simply add `grafana_ucloud.dashboard.id` to the final select field.
